README.md

```diff
@@ -171,7 +171,10 @@ Users can also install several drivers at once:
 Usage: `data-diff DB1_URI TABLE1_NAME DB2_URI TABLE2_NAME [OPTIONS]`
 
 See the [example command](#example-command-and-output) and the [sample
-connection strings](#supported-databases).
+connection strings](#supported-databases). 
+
+Note that for some databases, the arguments that you enter in the command line 
+may be case-sensitive. This is the case for the Snowflake schema and table names.
```

Comment:
As far as I know, only the schema name is case-sensitive.

We should handle case insensitive table names by default.

Reply:
@erezsh I thought we only handled case insensitivity on columns? Don't think we query the DB for the tables, do we?

Reply:
Hi @erezsh, [this](https://github.com/datafold/data-diff/blob/43c1c24bdac06e2753067d6671e23cb595596306/data_diff/databases/base.py#L157) is where I ran into the issue (you probably knew that).

In Snowflake, the following queries return 0 rows due to incorrect casing:
```
SELECT column_name, data_type, datetime_precision, numeric_precision, numeric_scale FROM information_schema.columns 
WHERE table_name = 'deal_facts' AND table_schema = 'DBT_LFOLSOM';
```
```
SELECT column_name, data_type, datetime_precision, numeric_precision, numeric_scale FROM information_schema.columns 
WHERE table_name = 'DEAL_FACTS' AND table_schema = 'dbt_lfolsom';
```
```
SELECT column_name, data_type, datetime_precision, numeric_precision, numeric_scale FROM information_schema.columns 
WHERE table_name = 'deal_facts' AND table_schema = 'dbt_lfolsom';
```
Only this version returns rows:
```
SELECT column_name, data_type, datetime_precision, numeric_precision, numeric_scale FROM information_schema.columns 
WHERE table_name = 'DEAL_FACTS' AND table_schema = 'DBT_LFOLSOM';
```

_Both_ `DEAL_FACTS` AND `DBT_LFOLSOM` have to be all-caps to match how they are named in the db. 

If I enter the command line arguments as lowercase, the query that is sent to Snowflake respects whatever case I entered, leading the query to fail.

So I believe both table and schema names are case-sensitive in Snowflake. 

I also think it is possible to ask Snowflake to store table or schema names with a particular case (lower, upper, or other), so it is unpredictable what should be sent to Snowflake. 

Personally, I like the idea of just telling the user in the README to be careful with casing, and maybe adding a similar warning to the error message, but that's just my preference; and as a user, I would consider that adequate/reasonable; and I wouldn't expect the maintainers to handle every possible casing scenario that could exist in a messy database.

Reply:
@erezsh for sure--my only point was that I thought it was a good idea to give the user a heads up in the README that at least for Snowflake, table and schema names are case sensitive. Are you ok with the update I made in the README to that effect?